<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-10-29T12:44:18 IST" name="Suite" finished-at="2024-10-29T12:44:27 IST" duration-ms="9004">
    <groups>
      <group name="Master">
        <method signature="TC001_AccountRegistrationTest.account_Registration()[pri:0, instance:testCases.TC001_AccountRegistrationTest@9597028]" name="account_Registration" class="testCases.TC001_AccountRegistrationTest"/>
      </group> <!-- Master -->
      <group name="Sanity">
        <method signature="TC001_AccountRegistrationTest.account_Registration()[pri:0, instance:testCases.TC001_AccountRegistrationTest@9597028]" name="account_Registration" class="testCases.TC001_AccountRegistrationTest"/>
      </group> <!-- Sanity -->
    </groups>
    <test started-at="2024-10-29T12:44:18 IST" name="Test" finished-at="2024-10-29T12:44:27 IST" duration-ms="9004">
      <class name="testCases.TC001_AccountRegistrationTest">
        <test-method is-config="true" signature="setup(java.lang.String,java.lang.String)[pri:0, instance:testCases.TC001_AccountRegistrationTest@9597028]" started-at="2024-10-29T12:44:19 IST" name="setup" finished-at="2024-10-29T12:44:24 IST" duration-ms="5438" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Windows]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="account_Registration()[pri:0, instance:testCases.TC001_AccountRegistrationTest@9597028]" started-at="2024-10-29T12:44:24 IST" name="account_Registration" finished-at="2024-10-29T12:44:27 IST" duration-ms="2634" status="FAIL">
          <exception class="java.lang.NullPointerException">
            <message>
              <![CDATA[Cannot invoke "java.lang.Throwable.getMessage()" because the return value of "org.testng.ITestResult.getThrowable()" is null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException: Cannot invoke "java.lang.Throwable.getMessage()" because the return value of "org.testng.ITestResult.getThrowable()" is null
at utilities.ExtentReportManager.onTestSuccess(ExtentReportManager.java:60)
at org.testng.internal.TestListenerHelper.runTestListeners(TestListenerHelper.java:100)
at org.testng.internal.TestInvoker.runTestResultListener(TestInvoker.java:219)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:656)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- account_Registration -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:testCases.TC001_AccountRegistrationTest@9597028]" started-at="2024-10-29T12:44:27 IST" name="tearDown" finished-at="2024-10-29T12:44:27 IST" duration-ms="326" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- testCases.TC001_AccountRegistrationTest -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
